app.rb

```diff
@@ -0,0 +1,75 @@
+ENV['RACK_ENV'] ||= 'development'
+
+require 'sinatra/base'
+require './config/data_mapper'
+require 'pry'
+
+
+class ChitterApp < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    @messages = Message.all
+    current_user
+    if logged_in?
+      erb :logged_in
+    else
+      erb :index
+    end
+  end
+
+  post '/post_msg' do
+    Message.create(
```

Comment:
If I understand well anyone is able to post a message a being logged in is not mandatory ?